OWML.ModHelper.Menus/ModMenu.cs

```diff
@@ -150,6 +170,20 @@ public IModTextInput AddTextInput(IModTextInput input, int index)
             AddInput(input, index);
             return input;
         }
+        public IModComboInput GetComboInput(string title)
+        {
+            return ComboInputs.FirstOrDefault(x => x.Title == title || x.Element.name == title);
+        }
+        public IModComboInput AddComboInput(IModComboInput input)
+        {
+            return AddComboInput(input, input.Index);
+        }
+        public IModComboInput AddComboInput(IModComboInput input, int index)
+        {
+            ComboInputs.Add(input);
+            AddInput(input, index);
+            return input;
+        }
```

Comment:
Newline between methods

Reply:
done

Reply:
Nope :P

Reply:
Still not done :P

Reply:
Should be done now (GiitHub cut this message off for some reason)